src/app/resources/rating-resources/resources-rating.component.ts

```diff
@@ -42,7 +42,7 @@ export class ResourcesRatingComponent implements OnChanges {
     return { rate: this.rating.userRating.rate || 0 };
   }
   get commentField() {
-    return { comment: this.rating.userRating.comment || '' };
+    return { comment: '' };
```

Comment:
this code is even removing old comment of user for that resource. Issue is not to have comment filled for new rating